sql/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'idea'
 apply plugin: 'java'
+apply plugin: 'de.thetaphi.forbiddenapis'
```

Comment:
I'll do a follow up PR for the other modules because there are more errors to be fixed.